### ISSUE TITLE
Stopped working on ubuntu 18.10

### ISSUE THREAD

**Issue by ARahim3** [maateen/avro#123]

**Comment by maateen** [comment]
It could be a bug. But I am out of Linux Desktop machine.

**Comment by mirashif** [comment]
Nope. It works fine. Just install with Gdebi package manager. Search in Ubuntu app store and install "Gdebi" and open avro .deb file with gdebi. It will install everything for you.

**Comment by maateen** [comment]
Unfortunately, I am out of any Linux machine. I am unable to help.

**Comment by gunnarhj** [comment]
I have met this problem in standard Ubuntu sessions with GNOME, while Avro works fine if I log in to a Unity session instead. So it might be a GNOME bug.

@ARahim3: Which is your desktop environment?

**Comment by gunnarhj** [comment]
Please see https://github.com/maateen/avro/issues/121#issuecomment-498834470
